### DISCUSSION TITLE
Is there a way to use a log scale?

### DISCUSSION THREAD

**Discussion by Ennoriel** [mhkeller/layercake#206] [Q&A] [answered]
I'd like to display one of my graph with a vertical linear scale and a horizontal linear scale. Is that possible? Would the quad tree still work?

Thanks for your awesome library!

**Comment by mhkeller** [comment, marked as answer]
You can pass in whatever scale you like using the `xScale`, `yScale`, `zScale` or `rScale` prop. Docs: https://layercake.graphics/guide#xscale
